update_gtn_posts.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+import feedparser
+
+feedparser.sanitizer._HTMLSanitizer.acceptable_elements.remove('a')
+feedparser.sanitizer._HTMLSanitizer.acceptable_elements.remove('img')
+
+feeds = feedparser.parse('https://training.galaxyproject.org/training-material/feed.xml')
+
+for entry in feeds.entries:
+    feed_title = entry['title']
+    feed_link = entry['link']
+    feed_blurb = entry['summary']
+    TEMPLATE = f"""---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+tags: [training, gtn-news]
+title: "{feed_title}"
+supporters:
+- denbi
+- elixir
```

Comment:
should we add Gallantries as supporter to these?

Reply:
sure, didn't pay attention to the supporters much, better?